### PR TITLE
Update changelog about local settings sync and ports view

### PR DESCRIPTION
**Pull request by loujaybee**
Introduce updates to documentation, so that we can deep link users who are part of UI experiments with VS Code. 

Adds section talking about the Local Settings Sync experiment:

image

Updates section on ports view

image

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2733"></a>

### CODE REVIEW

**Comment by loujaybee** [comment]
@akosyakov - let me know if you find any other pages in the documentation that is now outdated, we could maybe just address quickly in this PR so things aren't out-of-date.

**Comment by mustard-mh** [comment]
We need to add new ports view screenshot on this page too https://www.gitpod.io/docs/config-ports#specifying-port-names--descriptions

Should I file an issue?

**Comment by akosyakov** [comment]
> We need to add new ports view screenshot on this page too https://www.gitpod.io/docs/config-ports#specifying-port-names--descriptions

We should do it only after migration of all users.

> Should I file an issue?

Yes please.

**Review by akosyakov — APPROVED** [review]
lgtm

File: src/lib/contents/changelog/2022-08-31.md [hunk 1]
```diff
@@ -59,6 +60,13 @@ _Viewing the workspace and node performance in the Backend Control Center_
 
 We have been continuing to invest in our desktop editor experiences, including VS Code #[7184](https://github.com/gitpod-io/gitpod/issues/7184), with an emphasis on removing any barriers or friction in getting the VS Code editor open with a Gitpod workspace, establishing a stable connection via SSH, and improving the experience of restarting a timed out workspace.
 
+### Sync Local VS Code Extensions
+
+Currently, [VS Code - Remote SSH](https://code.visualstudio.com/docs/remote/ssh) does not automatically install local user extensions on remote machines. Gitpod extension prompts users to enable [VS Code Setting Sync](/docs/ides-and-editors/settings-sync#enabling-settings-sync-in-vs-code-desktop) with Gitpod to retrieve local extensions and automatically install extensions in your Gitpod workspace. You can enable, or disable syncing of local extensions via the `gitpod.remote.syncExtension` VS Code settings. If you have feedback, please add it to the GitHub issue: [#7906](https://github.com/gitpod-io/gitpod/issues/7906).
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-08-31.md:65]: **Question:** For my own clarity, for this to work you _must_ have VS Code Settings Sync enabled? But you can disable this one feature within VS Code Settings Sync? So you can have: 

1. Disabled gitpod Settings Sync AND disabled automatic Local Extension installation
2. Enabled gitpod Settings Sync AND disabled automatic Local Extension installation
3. Enabled gitpod Settings Sync AND enabled automatic Local Extension installation

Is that correct?

**Review by loujaybee — COMMENTED** [review]
Looks great, I can't approve since I'm the author. Dropped a wording suggestion, but otherwise the images look great, this should be perfect to link in support and the community about the reasoning behind the change. 

Thanks @akosyakov and @mustard-mh !

File: src/lib/contents/changelog/2022-08-31.md (hunk 1 above)
Comment (akosyakov) [reply src/lib/contents/changelog/2022-08-31.md:65]: yes i think so cc @jeanp413

**Comment by akosyakov** [comment]
@loujaybee I think you can apply suggestions and merge?

File: src/lib/contents/changelog/2022-08-31.md (hunk 1 above)
Comment (jeanp413) [reply src/lib/contents/changelog/2022-08-31.md:65]: yep that's correct

**Comment by akosyakov** [comment]
@loujaybee I approved it, merging?

**Comment by loujaybee** [comment]
@akosyakov yes, apologies. Had more build issues and needed to rebase, got distracted whilst waiting for the build. 

Thanks for merging!

**Pull request merged**